src/components/Menu/__stories__/index.stories.mdx

```diff
@@ -0,0 +1,177 @@
+import {
+  ArgsTable,
+  Canvas,
+  Description,
+  Meta,
+  Story,
+} from '@storybook/addon-docs/blocks'
+import Menu from '..'
+import { Box, Button, Icon, Modal, Touchable } from '../..'
+
+<Meta title="Components/Menu" component={Menu} />
+
+# Menu
+
+## Basic
+
+<Description>Menu</Description>
+
+<Canvas>
+  <Story name="Basic">
```

Comment:
```suggestion
  <Story name="Basic" height="300px">
```

to avoid this: 
![Screenshot 2021-06-14 at 11 38 01](https://user-images.githubusercontent.com/15812968/121872000-107ba000-cd05-11eb-82b2-eb2695bc0a69.png)